modules/cnf-du-annotating-workloads.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-annotating-workloads_{context}"]
+
+= Annotating workloads
+
+In order for a workload to be isolated to partitioned resources two annotations must be present. The containing namespace must be annotated with `workload.openshift.io/allowed` and the pod must be annotated with `target.workload.openshift.io`.
+
+Required OCP platform pods and namespaces  are already annotated. Supported day 2 operators need the namespace annotated.  If you want to add extra pods to the partition, such as any user-added catalog sources, both the namespace and pod must be annotated on creation.
```

Comment:
I would remove the sentence "Supported day 2 operators need the namespace annotated". Above we already state that the namespace and pod must be annotated. There is nothing unique about supported day-2 operators needing the namespace, it is true for any pod needing to be partitioned. The following sentence conveys this well. Perhaps add day-2 to it, eg: 
such as supported day-2 operators or any user-added catalog sources

Reply:
Comment accepted.